README.md

```diff
@@ -158,9 +158,8 @@ License
 
 GPLv3
 
-Author Information
+Authors Information
 ------------------
 
-Eric Baum, 2018
-
-https://github.com/ericbaum
+1. Eric Baum, 2018 - https://github.com/ericbaum
+2. Revomatico - http://revomatico.com
```

Comment:
;) It was really missing as you are doing all the job right now

Reply:
Well, thank you for starting this
Keeping up with the rook updates is not a big deal, but we do it since we are actually using this role to deploy and upgrade ceph cluster in a simple manner.